usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -495,6 +494,129 @@ def load_file_a(submission_attributes, appropriation_data, db_cursor):
         submission_attributes.submission_id)
 
 
+def get_file_b(submission_attributes, db_cursor):
+    """
+    Get broker File B data for a specific submission.
+    This function was added as a workaround for the fact that a few agencies
+    (two, as of April, 2017: DOI and ACHP) submit multiple File B records
+    for the same object class. These "dupes", come in as the same 4 digit object
+    class code but with one of the direct reimbursable flags set to NULL.
+
+    From our perspective, this is a duplicate, because we get our D/R info from
+    the 1st digit of the object class when it's four digits.
+
+    Thus, this function examines the File B data for a given submission. If
+    it has the issue of "duplicate" object classes, it will squash the
+    offending records together so that all financial totals are reporting
+    as a single object class/program activity/TAS record as expected.
+
+    If the broker validations change to prohibit this pattern in the data,
+    this intervening function will no longer be necessary, we can go back to
+    selecting * from the broker's File B data.
+
+    Args:
+        submission_attrbitues: submission object currently being loaded
+        db_cursor: db connection info
+    """
+    submission_id = submission_attributes.broker_submission_id
+
+    # does this file B have the dupe object class edge case?
+    check_dupe_oc = (
+        'SELECT count(*) '
```

Comment:
A possible performance improvement would be to replace this count where we don't actually care about the number with an exists: https://blog.jooq.org/2016/09/14/avoid-using-count-in-sql-when-you-could-use-exists/

The first step would be to determine if this is a useful improvement or not - if the query is not much of a choke point and/or will hopefully be cleansed with fire soon, it's not worth worrying about, and in any case shouldn't hold up this commit; it could be noted for future investigation.